### PR TITLE
dont set ACLs if there are no ACLs to change

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: spec/acceptance/updating_posix_acl_spec.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+describe 'posix_acl' do
```
Comment (ekohl) [inline spec/acceptance/updating_posix_acl_spec.rb:5]: This now overlaps with the other test. I don't know if that works well.

**Review by ekohl — COMMENTED** [review]
I wonder if you can verify the changes made with assertions.

File: spec/acceptance/updating_posix_acl_spec.rb (hunk 1 above)
Comment (bastelfreak) [reply spec/acceptance/updating_posix_acl_spec.rb:5]: ah. I can move it to the existing file.

**Comment by bastelfreak** [comment]
I will try to debug this tomorrow. a customer reported it. This bug is already present in our 1.1.0 release.

File: spec/acceptance/updating_posix_acl_spec.rb (hunk 1 above)
Comment (ekohl) [reply spec/acceptance/updating_posix_acl_spec.rb:5]: The alternative is to use a new name, but either works.

**Comment by bastelfreak** [comment]
https://github.com/voxpupuli/puppet-posix_acl/pull/109 has the same test like this PR. https://github.com/voxpupuli/puppet-posix_acl/pull/108/commits/01a20309b0d4cf2487e3dcdb5ef61dc794dc47f6 fixes the error

**Comment by bastelfreak** [comment]
I'm pretty sure this is a valid bugfix and won't introduce a regression, but not 100% sure :D

**Comment by bastelfreak** [comment]
I would like to rebase it after https://github.com/voxpupuli/puppet-posix_acl/pull/111 got merged

**Review by smortex — COMMENTED** [review]
Not a user of POSIX ACLs but at least the "logic" looks good :shrug:

**Comment by bastelfreak** [comment]
Fixes https://github.com/voxpupuli/puppet-posix_acl/issues/112

**Pull request merged**